_build/buildPDF.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# this generates A PDF from a HTML file
+
+if [ ! -x _build/build.sh ] ; then
+  echo "ERROR: script must be run in root of docs dir"
```

Comment:
I'm not sure this is the correct error message after checking the file exists and executable. Possible solution to avoid this error is to get the absolute script path, i.e. ```$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )```